tests/archive/tests_management_command.py

```diff
@@ -355,6 +355,92 @@ def test_notify_inactive_jobseekers_on_eligibility_diagnosis_expiration_date(sel
             ordered=False,
         )
 
+    def test_notify_inactive_jobseekers_on_job_application_dates(self):
+        old_job_application_kwargs = {
+            "job_seeker__joined_days_ago": DAYS_OF_INACTIVITY,
+            "created_at": timezone.now() - relativedelta(days=DAYS_OF_INACTIVITY),
+            "eligibility_diagnosis": None,
+        }
+        recent_job_application_kwargs = old_job_application_kwargs.copy()
+        recent_job_application_kwargs["created_at"] = recent_job_application_kwargs["created_at"] + relativedelta(
+            days=1
+        )
+        transition_kwargs = {
+            "from_state": JobApplicationState.NEW,
+            "to_state": JobApplicationState.PROCESSING,
+        }
+
+        # old job application without transition
+        old_job_application_wo_transition = JobApplicationFactory(**old_job_application_kwargs)
+
+        # old job application with old transition
+        old_job_application_and_transition = JobApplicationFactory(**old_job_application_kwargs)
```

Comment:
```suggestion
        old_job_application_with_old_log = JobApplicationFactory(**old_job_application_kwargs)
```
`and` -> `with`, et ajouter le `old` (en symétrie du `recent` pour les autres candidatures)